brianlong/www.validators.app
# 187088712 archive pings

#### What's this PR do?
- create archive of ping things and ping thing stats

#### How should this be manually tested?
- run migrations
- run scripts in script/archives
- check if there are no run errors
- check if the records are copied/moved correctly

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/187088712)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [Y] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
